datadog/tests/resource_datadog_logs_metric_test.go

```diff
@@ -82,8 +82,9 @@ func testAccCheckDatadogLogsMetricConfigBasic(uniq string) string {
         resource "datadog_logs_metric" "testing_logs_metric" {
 			name = "%s"
 			compute {
-				aggregation_type = "distribution"
-				path             = "@duration"
+				aggregation_type    = "distribution"
+				path                = "@duration"
+				include_percentiles = false
```

Comment:
Can we update the test `TestAccDatadogLogsMetric_Basic` as well to check the terraform resource state contain the `include_percentiles` value ?
Like https://github.com/DataDog/terraform-provider-datadog/blob/fd929126ea0cfd8e86605d3d12fc63be4a040e35/datadog/tests/resource_datadog_logs_metric_test.go#L72

Reply:
🤦 I removed it and forgot to put it back. Updated here 👌 (https://github.com/DataDog/terraform-provider-datadog/pull/1645/commits/e3881747942922254a069690327ce144c0716ea4)

Reply:
Thanks ! 🙇‍♂️